README.md

```diff
@@ -5,6 +5,22 @@ This repository houses tools built on the GPU-accelerated simulation capabilitie
 * massively-parallelized simulation.
 
 ## Installation
+### Via `pip`
+There's currently no official pypi release, but you can still install this package via `pip`. We recommend installing it in a conda environment with Cuda 11.8 support.
+```
+# all development dependencies
+pip install "ambersim[all] @ git+https://github.com/Caltech-AMBER/ambersim.git"
```

Comment:
When I run this on a fresh env, I get a massive dependency conflict (see below). I suspect this has to do with not specifying the right place to get `jax[cuda11-local]`?

```
...
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.6 does not provide the extra 'cuda11-local'
  Downloading jax-0.4.5.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 51.5 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.5 does not provide the extra 'cuda11-local'
  Downloading jax-0.4.4.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 51.8 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.4 does not provide the extra 'cuda11-local'
  Downloading jax-0.4.3.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 51.2 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.3 does not provide the extra 'cuda11-local'
  Downloading jax-0.4.2.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 52.3 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.2 does not provide the extra 'cuda11-local'
  Downloading jax-0.4.1.tar.gz (1.2 MB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.2/1.2 MB 51.7 MB/s eta 0:00:00
  Preparing metadata (setup.py) ... done
WARNING: jax 0.4.1 does not provide the extra 'cuda11-local'
ERROR: Cannot install ambersim, ambersim[all]==0.0.1, jax[cuda11-local]==0.4.1, jax[cuda11-local]==0.4.10, jax[cuda11-local]==0.4.11, jax[cuda11-local]==0.4.12, jax[cuda11-local]==0.4.13, jax[cuda11-local]==0.4.14, jax[cuda11-local]==0.4.16, jax[cuda11-local]==0.4.17, jax[cuda11-local]==0.4.18, jax[cuda11-local]==0.4.19, jax[cuda11-local]==0.4.2, jax[cuda11-local]==0.4.20, jax[cuda11-local]==0.4.3, jax[cuda11-local]==0.4.4, jax[cuda11-local]==0.4.5, jax[cuda11-local]==0.4.6, jax[cuda11-local]==0.4.7, jax[cuda11-local]==0.4.8 and jax[cuda11-local]==0.4.9 because these package versions have conflicting dependencies.

The conflict is caused by:
    ambersim[all] 0.0.1 depends on jaxlib>=0.4.1
    brax 0.9.3 depends on jaxlib>=0.4.6
    jax[cuda11-local] 0.4.20 depends on jaxlib==0.4.20+cuda11.cudnn86; extra == "cuda11_local"
    ambersim[all] 0.0.1 depends on jaxlib>=0.4.1
    brax 0.9.3 depends on jaxlib>=0.4.6
    jax[cuda11-local] 0.4.19 depends on jaxlib==0.4.19+cuda11.cudnn86; extra == "cuda11_local"
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.18 depends on jax 0.4.18 (from https://files.pythonhosted.org/packages/b5/5b/5131520dd9a384a640399e5efe4324fdee9e8a48685a33d08eb47140ccc3/jax-0.4.18-py3-none-any.whl (from https://pypi.org/simple/jax/) (requires-python:>=3.9))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.17 depends on jax 0.4.17 (from https://files.pythonhosted.org/packages/fb/b5/c1222696f99a51fe932243924d97c8ce89e654a0c571670cbdd3cc6fef83/jax-0.4.17-py3-none-any.whl (from https://pypi.org/simple/jax/) (requires-python:>=3.9))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.16 depends on jax 0.4.16 (from https://files.pythonhosted.org/packages/2f/7e/052564673066fcb143a7d096232790d037554b323f0cf0e7c2af3a60ca88/jax-0.4.16-py3-none-any.whl (from https://pypi.org/simple/jax/) (requires-python:>=3.9))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.14 depends on jax 0.4.14 (from https://files.pythonhosted.org/packages/65/ce/e3a6e8669de6ff37b44b1f801c33c10dcdc05548ee5ded30c0327eb09c93/jax-0.4.14.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.9))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.13 depends on jax 0.4.13 (from https://files.pythonhosted.org/packages/1e/f2/03643b515aede51a812608e3fe37cbc30426241e512bb0fb8546c36ddd5b/jax-0.4.13.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.12 depends on jax 0.4.12 (from https://files.pythonhosted.org/packages/91/06/00616821a40137a96ab8192473687dfbdc5dc87f6aa83c4f36630bc3f11f/jax-0.4.12.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.11 depends on jax 0.4.11 (from https://files.pythonhosted.org/packages/97/7f/f0ba64f1aedf032cf1d376fa85275a4c7979aefba7ca663432f5d93488d2/jax-0.4.11.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.10 depends on jax 0.4.10 (from https://files.pythonhosted.org/packages/13/71/679a0ef6821a5fc71008e044a81e5c2da4fdee3a5aca9fea9fcfe594f163/jax-0.4.10.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.9 depends on jax 0.4.9 (from https://files.pythonhosted.org/packages/08/fd/9af49f926cbbc686977c41cdee21669288500b5dbf2e8eeb6b50eacfb00e/jax-0.4.9.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.8 depends on jax 0.4.8 (from https://files.pythonhosted.org/packages/fe/58/1641614c17fcd7293d250c2cad48011baa1ecef4f109ce2ea027aa8e4898/jax-0.4.8.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.7 depends on jax 0.4.7 (from https://files.pythonhosted.org/packages/6d/c7/798a917fa8e5718de5a999883e93cf77738c2b2fb474c1796de0190918ee/jax-0.4.7.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.6 depends on jax 0.4.6 (from https://files.pythonhosted.org/packages/96/76/cb0a6f68915744d3098d09161745f69f33193d83edd18eb1ed3c372fc5e8/jax-0.4.6.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.5 depends on jax 0.4.5 (from https://files.pythonhosted.org/packages/46/d8/0ff2f887837dff08c1fe8f2fdd2781a9982e9486e3ab48177c737725d1f9/jax-0.4.5.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.4 depends on jax 0.4.4 (from https://files.pythonhosted.org/packages/29/8a/4b43070e1a5951e2c4bece5efb6046f3ef87c427764c6b7ae20159b44bba/jax-0.4.4.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.3 depends on jax 0.4.3 (from https://files.pythonhosted.org/packages/96/f2/cfbefdd74bb04e02c8a0599332c22db9adbdf2b13775de3ce9b0445bdf18/jax-0.4.3.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.2 depends on jax 0.4.2 (from https://files.pythonhosted.org/packages/c9/cd/5d77845520ea6f3c96b6121ddba7068f9f1b789f4a534005a29cce829804/jax-0.4.2.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))
    ambersim[all] 0.0.1 depends on jax>=0.4.1
    brax 0.9.3 depends on jax>=0.4.6
    flax 0.7.5 depends on jax>=0.4.19
    jax[cuda11-local] 0.4.1 depends on jax 0.4.1 (from https://files.pythonhosted.org/packages/e5/aa/3adbd63fe645ee4fbc0daeba9423b21970929145bc63cbf2d813b9d150f5/jax-0.4.1.tar.gz (from https://pypi.org/simple/jax/) (requires-python:>=3.8))

To fix this you could try to:
1. loosen the range of package versions you've specified
2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts
```

Reply:
ah yes sorry. the correct command should be
```
pip install "ambersim[all] @ git+https://github.com/Caltech-AMBER/ambersim.git" --find-links https://storage.googleapis.com/jax-releases/jax_cuda_releases.html
```